tests/benchmarks/multicore/idle/testcase.yaml

```diff
@@ -0,0 +1,49 @@
+common:
+  build_only: true
```

Comment:
Please do not use build_only,
rather define expected regex on console, eg:
  harness: console
  harness_config:
    type: multi_line
    ordered: true
    regex:
      - "Multicore idle test on"
      - "Multicore idle test iteration 0"
      - "Multicore idle test iteration 1"